libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,244 @@
+import cl from 'clsx';
+import classes from './Spinner.module.scss';
+
+export interface SpinnerProps {
+  size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
+  variant?: 'default' | 'inverted';
+  label?: string;
+}
+
+export function Spinner({
+  size = 'medium',
+  variant = 'default',
+  label,
+}: SpinnerProps) {
+  let PathStrokeSubtle = '';
+  let PathStrokeAction = '';
+  let Eclipse = null;
+  let Labelsize: 'small' | 'medium' = 'medium';
+
+  switch (variant) {
+    case 'default':
+      PathStrokeSubtle = 'var(--px-color-border-subtle)';
+      PathStrokeAction = 'var(--px-color-border-action)';
+      break;
+    case 'inverted':
+      PathStrokeSubtle = 'var(--px-color-border-subtle)';
+      PathStrokeAction = 'var(--px-color-border-on-inverted)';
+      break;
+    default:
+      PathStrokeSubtle = 'var(--px-color-border-subtle)';
+      PathStrokeAction = 'var(--px-color-border-action)';
+      break;
+  }
+  switch (size) {
```

Comment:
Theres a lot of code being repeated in the SVGs here. I'd prefer only changing some variables and then having one svg part, but I'll leave the decision here to you :)

Reply:
Changes done.